### ISSUE TITLE
AdvancedSlider should have an option to suffix and prefix min/max values, and not require a "+" suffix on max value

### ISSUE THREAD

**Issue by vchelaru** [enisn/Xamarin.Forms.InputKit#69]
**Is your feature request related to a problem? Please describe.**
Currently the min and max values cannot have prefixes and suffixes like the Value. Since the min and man are using the same units as the current value, they should optionally have prefixes and suffixes.

Also, I find it very strange that the max value has a "+" at the end. I don't want that in my app, and if users could add prefixes and suffixes, they could optionally add a "+"

**Describe the solution you'd like**

I would like to see the following values added to Advanced Slider:

```
public string MinValuePrefix { get; set;}
public string MaxValuePrefix { get; set;}
public string MinValueSuffix { get; set;}
public string MaxValueSuffix { get; set;}
```

**Describe alternatives you've considered**
I have considered using the regular Xamarin slider, but it doesn' tlook like it supports a `StepValue` which is very important for my app.

I will likely not display the min and max values and instead create a grid where I show these values with labels.

**Additional context**
<None>

**Comment by enisn** [comment]
That problem solved at **v2.3.8** and released.

@vchelaru ,  can you try new version and notify result if worked or not?

<hr /> 

Release notes are [here](https://github.com/enisn/Xamarin.Forms.InputKit/releases/tag/v.2.3.8)
NuGet Package is [here](https://www.nuget.org/packages/Xamarin.Forms.InputKit/2.3.8)

**Comment by vchelaru** [comment]
Thanks so much it works great!
